sdk/monitor/monitor-query/TROUBLESHOOTING.md

```diff
@@ -1,5 +1,10 @@
 # Troubleshooting Azure Monitor Query client library issues
 
+> Deprecated ⚠️: This package is deprecated. For new development, use:
```

Comment:
I think it is overkill to put the notice here.

Reply:
removed